### DISCUSSION TITLE
Android Capacitor plugin with submodules

### DISCUSSION THREAD

**Discussion by gmbett** [ionic-team/capacitor#5243] [Q&A]
Hi,

I'm trying to build a capacitor in which the Android code uses internally sub-modules, e.g: app-core

My main problem is that when I do a `npm install [path to plugin]`, the main project doesn't work properly because it cannot find the submodule dependency inside the plugin.

Do you know how to solve this?

**Comment by renatojobal** [comment]
@gmbett Did you find any solution? I am facing the same problem here.
